berkeley-function-call-leaderboard/bfcl/eval_checker/multi_turn_eval/func_source_code/message_api.py

```diff
@@ -67,6 +71,7 @@ def __init__(self):
         }
         self.message_count: int = 0  # useless(?)
         self.current_user: Optional[str] = None
+        self._random: random.Random = random.Random(200191)
```

Comment:
We already have a `self._random` initialized in the `_load_scenario` method, why do we need to do it in the `__init__` again?

Reply:
It raises an error when calling the function generate_id() (in send_message)